chainerx_cc/chainerx/routines/math.cc

```diff
@@ -546,16 +555,23 @@ Array Log(const Array& x) {
 }
 
 Array LogSumExp(const Array& x, const OptionalAxes& axis, bool keepdims) {
+    // TODO(imanishi): Support dtype promotion for arithmetics functions
```

Comment:
What does this comment mean?

Reply:
It means that these lines should be fixed after #6432 is merged.

Reply:
Are you referring to the last `+`?
If so, place the comment before that line.
Make the comment more verbose.